subprojects/packagefiles/raylib/meson.build

```diff
@@ -0,0 +1,94 @@
+project(
+  'raylib',
+  'c',
+  version: '5.5',
+  license: 'zlib',
+  default_options: ['c_std=none'],
+  meson_version: '>=1.1.0',
```

Comment:
It's not worth a 1.1.0 minimum solely for `meson.options`.  Let's call it `meson_options.txt`.

Reply:
Sure 👍